utils/browser_history.tsx

```diff
@@ -27,7 +37,7 @@ window.addEventListener('message', ({origin, data: {type, message = {}} = {}} =
 
 export const browserHistory = {
     ...b,
-    push: (path, ...args) => {
+    push: (path: any, ...args: string[]) => {
         if (isDesktop) {
```

Comment:
The suggestion will delete the wrong line just as a heads up :P

As for `path`, it should always be `string`, I'm not sure where it can be `Record<string, string>`, unless the `history` module defines it as such?

Reply:
Also, there's no guarantee the other `args` are `string`, maybe we should use `unknown` there?

Reply:
we are accessing a pathname property from path which can't happen if path is string.
so whether it can be string or Record<something, something>

Reply:
If it's always one of those two, can we be more explicit then, ie. type it as `string | { pathname: string }`?